frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts

```diff
@@ -999,3 +987,26 @@ function deduplicatePerformanceEvents(events: PerformanceEvent[]): PerformanceEv
         })
         .reverse()
 }
+
+/**
+ * If we have paint events we should add them to the appropriate navigation event
+ * this makes it easier to draw performance cards for navigation events
+ */
+function matchPaintEvents(performanceEvents: PerformanceEvent[]): PerformanceEvent[] {
+    // KLUDGE: this assumes that the input is sorted by timestamp and relies on the identity of the events to mutate them
```

Comment:
Is there any reason we wouldn't do the sorting inside this function to make sorting a requirement

Reply:
yeah, I was torn... was too lazy to find out if it is cheap to sort an already sorted thing....

Reply:
turns out it is slightly faster to sort an already sorted list of timestamps... but tbh despite the wastefulness of sorting multiple times a quick benchmark managed 38k sort operations per second for a list of 1000 items so I'm completely worrying unnecessarily 🤣